source/Ship.cpp

```diff
@@ -1439,18 +1439,18 @@ int Ship::Scan()
 	if(government->IsPlayer() || (target->GetGovernment()->IsPlayer() && activeScanning))
 		Audio::Play(Audio::Get("scan"), Position());
 	
-	if(startedScanning && government->IsPlayer())
+	if(startedScanning && government->IsPlayer() && !target->GetGovernment()->IsPlayer())
```

Comment:
Sure. But referring to your own flagship as an "Escort ship" is just... well... incorrect. :P